portality/static/doaj/js/available_facetviews/admin.editorgroups.facetview.js

```diff
@@ -21,7 +21,7 @@ jQuery(document).ready(function($) {
             {'display':'Created Date','field':'created_date'},
             {'display':'Last Modified Date','field':'last_updated'},
             {'display':'Editor ID','field':'editor'},
```

Comment:
Editor account names tend to be one word, so I'm seeing it ordering as we expect on the live site. I can change it for completeness, if you like.